app/src/common/services/role/roleService.spec.ts

```diff
@@ -0,0 +1,342 @@
+namespace common.services.user {
+
+    describe('UserService', () => {
+
+        let userService:UserService;
+        let $httpBackend:ng.IHttpBackendService;
+        let authService:NgJwtAuth.NgJwtAuthService;
+        let ngRestAdapter:NgRestAdapter.NgRestAdapterService;
+        let $mdDialog:ng.material.IDialogService;
+        let $timeout:ng.ITimeoutService;
+        let $rootScope:ng.IRootScopeService;
+
+        beforeEach(()=> {
+
+            module('app');
+
+            inject((_$httpBackend_, _userService_, _ngJwtAuthService_, _ngRestAdapter_, _$mdDialog_, _$timeout_, _$rootScope_) => {
+
+                if (!userService) { //dont rebind, so each test gets the singleton
+                    $httpBackend = _$httpBackend_;
+                    userService = _userService_;
+                    authService = _ngJwtAuthService_;
+                    ngRestAdapter = _ngRestAdapter_;
+                    $mdDialog = _$mdDialog_;
+                    $timeout = _$timeout_;
+                    $rootScope = _$rootScope_;
+                }
+            });
+
+        });
+
+        afterEach(() => {
+            $httpBackend.verifyNoOutstandingExpectation();
+            $httpBackend.verifyNoOutstandingRequest();
+        });
+
+        describe('Initialisation', () => {
```

Comment:
Are these tests just copied for user service? Haven't been updated to tests for role service yet?